src/service/performance-impl.js

```diff
@@ -353,12 +348,7 @@ export class Performance {
           this.layoutShifts_.push(entry);
         }
       } else if (entry.entryType === 'largest-contentful-paint') {
-        if (entry.loadTime) {
-          this.largestContentfulPaintLoadTime_ = entry.loadTime;
-        }
-        if (entry.renderTime) {
-          this.largestContentfulPaintRenderTime_ = entry.renderTime;
-        }
+        this.largestContentfulPaint_ = entry.startTime;
```

Comment:
Do all browsers support `startTime`?

Reply:
[Yes](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry/startTime), this is also how the [web-vitals](https://github.com/GoogleChrome/web-vitals/blob/cfe7536dc8f13e3cbbbf48436328f52399dd6725/src/getLCP.ts#L32-L45) package implements it.

Reply:
@jridgewell: I've updated `TICKEVENTS.md`